api/v1beta1/packetcluster_webhook.go

```diff
@@ -64,17 +81,26 @@ func (c *PacketCluster) ValidateUpdate(oldRaw runtime.Object) error {
 		)
 	}
 
-	if !reflect.DeepEqual(c.Spec.Facility, old.Spec.Facility) {
+	if !reflect.DeepEqual(c.Spec.VIPManager, old.Spec.VIPManager) {
 		allErrs = append(allErrs,
-			field.Invalid(field.NewPath("spec", "Facility"),
-				c.Spec.Facility, "field is immutable"),
+			field.Invalid(field.NewPath("spec", "VIPManager"),
+				c.Spec.VIPManager, "field is immutable"),
 		)
 	}
 
-	if !reflect.DeepEqual(c.Spec.VIPManager, old.Spec.VIPManager) {
+	// Must have at least Metro or Facility specified
+	if len(c.Spec.Facility) == 0 && len(c.Spec.Metro) == 0 {
 		allErrs = append(allErrs,
-			field.Invalid(field.NewPath("spec", "VIPManager"),
-				c.Spec.VIPManager, "field is immutable"),
+			field.Invalid(field.NewPath("spec", "Metro"),
+				c.Spec.Metro, "Metro is required when Facility is not set"),
```

Comment:
Same comment here as above; I think it's better to say `metro is required` than to tell a user who forgot both fields that they could use either one.